tests/integration/run_modes/test_skip.py

```diff
@@ -174,32 +176,25 @@ async def test_doesnt_release_held_tasks(
 
     """
     one_conf['runtime'] = {'one': {'run mode': 'skip'}}
-    schd = scheduler(flow(one_conf), run_mode='live')
-    async with start(schd) as log:
+    schd = scheduler(flow(one_conf), run_mode='live', paused_start=False)
+    async with run(schd) as log:
         itask = schd.pool.get_tasks()[0]
         msg = 'held tasks shoudn\'t {}'
 
         # Set task to held and check submission in skip mode doesn't happen:
         itask.state.is_held = True
-        schd.task_job_mgr.submit_task_jobs(
-            schd.workflow,
-            [itask],
-            schd.server.curve_auth,
-            schd.server.client_pub_key_dir,
-            run_mode=schd.get_run_mode()
-        )
+
+        # Relinquish contol to the main loop.
+        await asyncio.sleep(1)
```

Comment:
Try using `schd.release_queued_tasks` (should be able to remove the sleep and switch back from `run` to start`).